INSTALLATION.md

```diff
@@ -338,9 +338,27 @@ GraphQL endpoint for handling `subscriptions` is this:
 
 ## For Mobile App Developers
 
-The Organization URL for Talawa mobile app developers to use is:
+The Organization URL for Talawa mobile app developers will depend upon the device on which Mobile app is installed.
 
-       http://localhost:4000/graphql/
+- If Talawa Mobile App is installed on Android Virtual Device (AVD): Use the following URL:
+```
+    http://10.0.2.2:4000/graphql
+```
+- If Talawa Mobile App is installed on a Read Android Device: Use the following method to get URL.
+    - Open Command Prompt in Windows, or Terminal in Linux/OSX
+    - Enter ```ipconfig``` (For Windows Users) or ```ifconfig``` (For Linux/OSX Users)
+    - Your Mobile and Computer (On which API server is running) must be on same Wifi Network. Use Mobile Hotspot to connect your computer to internet in case you don't have access to a Wifi Router.
+    - Search for the ```Wireless LAN adapter Wi-Fi:``` and then copy ```IPv4 Address```, like in image below:
+    - ![image](https://user-images.githubusercontent.com/94157520/223067277-d1e40093-a018-479c-bf3c-07749fbb75b9.png)
```

Comment:
User an image on the repo. We want to make sure that this works even when the person is working completely offline.

Reply:
Done!